app/Http/Controllers/HomeController.php

```diff
@@ -25,4 +26,9 @@ public function index()
     {
         return view('home');
     }
+
+    public function showRecords(): View
```

Comment:
**Additional feature**: New getRecords method to show a view with imported records.